compiler/lib/js_output.ml

```diff
@@ -853,9 +852,9 @@ struct
     | EAssignTarget t -> (
         let property f p =
           match p with
-          | TargetPropertyId (Prop_and_ident id, None) -> ident f id
+          | TargetPropertyId (Prop_and_ident id, None) -> ident f ~kind:`Reference id
```

Comment:
This could be an implicit variable declaration, indeed. But this is bad practice and the identifier should have been declared somewhere else. The expression `EVar v` can also declare a variable, by the way, when it is on the right-hand side of an assignment operator.
I'm not sure we want to have larger source maps just to support this.

Reply:
Ok.